api/app/controllers/api/v1/instructors_controller.rb

```diff
@@ -1,61 +1,52 @@
 # frozen_string_literal: true
 
 module Api::V1
-  # Controller for Instructors
-  class InstructorsController < ApplicationController
-    before_action :set_instructor, only: %i[show update destroy]
-
-    # GET /instructors
-    def index
-      @instructors = Instructor.order(:id)
-
-      render json: @instructors
-    end
-
-    # GET /instructors/1
-    def show
-      render json: @instructor
-    end
-
-    # POST /instructors
-    def create
-      @instructor = Instructor.new(instructor_params)
-
-      if @instructor.save
-        render json: @instructor, status: :created
-      else
-        render json: @instructor.errors, status: :unprocessable_entity
-      end
-    end
-
-    # PATCH/PUT /instructors/1
-    def update
-      if @instructor.update(instructor_params)
-        render json: @instructor
-      else
-        render json: @instructor.errors, status: :unprocessable_entity
-      end
-    end
-
-    # DELETE /instructors/1
-    def destroy
-      if @instructor.destroy
-        head :no_content, status: :ok
-      else
-        render json: @position.errors, status: :unprocessable_entity
-      end
-    end
+    # Controller for Instructors
+    class InstructorsController < ApplicationController
+
+        # GET /instructors
+        def index
+            if Position.exists?(id: params[:position_id])
+                render json: { status: 'success', message: '', payload: instructors_by_position }
+            else
+                render json: { status: 'error', message: 'Invalid position_id', payload: {} }
```

Comment:
What if `:position_id` isn't provided because this is used from a different route?

Reply:
I've made a new route `GET /api/v1/instructors` for the `index` function. Is this what you mean?

Reply:
Yep, that's what I mean.